### ISSUE TITLE
Confirm clearing search history

### ISSUE THREAD

**Issue by avidseeker** [organicmaps/organicmaps#8573]
Sometimes search history contains important landmarks and geographic location, and a simple touch by mistake could clear up all search history. I suggest adding a confirmation dialog before clearing up all search entries.

**Comment by biodranik** [comment]
Is there a way to reduce the chance of accidental click without adding a confirmation dialog?

**Comment by avidseeker** [comment]
Losing a recent important location by mistake is worse than the inconvenience of another prompt confirmation.

**Comment by avidseeker** [comment]
You're right. I was having in mind the behavior of that maps app that appends location entries opened from links. But it looks like this is not in OrganicMaps: (https://github.com/organicmaps/organicmaps/issues/8576). If that feature is implemented, then this issue would be more relevant.
